app/src/main/java/org/mozilla/fenix/ext/BrowserState.kt

```diff
@@ -11,10 +11,22 @@ import mozilla.components.browser.state.state.TabSessionState
 import mozilla.components.feature.tabs.ext.hasMediaPlayed
 import org.mozilla.fenix.home.recenttabs.RecentTab
 import org.mozilla.fenix.tabstray.browser.TabGroup
-import org.mozilla.fenix.tabstray.browser.maxActiveTime
 import org.mozilla.fenix.tabstray.ext.isNormalTabActiveWithSearchTerm
+import org.mozilla.fenix.tabstray.ext.isNormalTabInactive
+import org.mozilla.fenix.utils.Settings
+import java.util.concurrent.TimeUnit
 import kotlin.math.max
 
+/**
+ * The time until which a tab is considered in-active (in days).
+ */
+const val DEFAULT_ACTIVE_DAYS = 14L
```

Comment:
Thank you for moving these constants out of the UI class. I kept forgetting to do that!